src/main/resources/Competition.sql

```diff
@@ -0,0 +1,98 @@
+CREATE TABLE City
+(
+	id SERIAL PRIMARY KEY,
+	name_city VARCHAR(255) NOT NULL
```

Comment:
(id, name). Everybody understand that field 'name' inside table 'City' means 'name of the city'
Please, check all your tables and rename fields